README.md

```diff
@@ -89,7 +89,7 @@ Clone this repository to your desired folder:
 
 ## 👥 Authors <a name="authors"></a>
 :bust_in_silhouette: *Mahdi Noori*
-- GitHub: [mahdinoori](https://github.com/mahdinoori2000)
+- GitHub: [Mahdinoori](https://github.com/mahdinoori2000)
 - LinkedIn: [Mahdi Noori](https://www.linkedin.com/in/mahdi-noori-hc201/)
 
```

Comment:
### USER SPEC

- Hey, a test examples fails on the user validation, Please, kindly fix it below:

![image](https://github.com/mahdinoori2000/blog-app/assets/59397159/8e4b3d56-43c8-4eec-9fb2-76dec415cfdf)


- The same issue on the post, kindly fix it

![image](https://github.com/mahdinoori2000/blog-app/assets/59397159/37e984c4-49e8-4a9b-b342-8cdf00cdfb2a)


- The same issue of the likes model. kindly fix

![image](https://github.com/mahdinoori2000/blog-app/assets/59397159/62f1a821-53df-4a9c-a38c-099972602293)


- The same issue for the comment, Please, kindly fix it

![image](https://github.com/mahdinoori2000/blog-app/assets/59397159/6abb7212-4b10-43cd-912c-67f99e6a10d2)

*[OPTIONAL]* Please, kindly move the test for your model validation into a model folder, since the test is written for the model's validation. this will make your test more readable in the folder structure.